index.html

```diff
@@ -0,0 +1,234 @@
+                <rect x="0.772461" y="12.7279" width="18" height="18" transform="rotate(-45 0.772461 12.7279)" fill="#FF6B00"/>
+              </svg>
+              <h2>Language</h2>
+            </div>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Css</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="1f" height="18" fill="#FF6B00"/>
+              </svg>    
+              <h2>Frameworks</h2>
+            </div>
+            
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"/>
+              </svg>     
+              <h2>Skills</h2>
+            </div>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>         
+ 
+
+    </section>
+    <!-- contact-form section. -->
+    <section class="contact-section">
+            <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+              <div>
+                <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
+                <input class="form-input" type="text" name="name" placeholder="Last name" maxlength="30" required>
+              </div>
+                <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30" required>
+                <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+                <input class="send" type="submit" value="Get in touch">
+            </form>
+    </section>
+    <footer>
+        <hr>
+          <ul class="footer-container">
+            <li><a><img src="images/github.png" alt="github"/></a></li>
+            <li><a><img src="images/linkedin.png" alt="linkedin"/></a></li>
+            <li><a><img src="images/angel list.png" alt="angel-list"/></a></li>
+            <li><a><img src="images/twitter.png" alt="twitter"/></a></li>
+            <li><a><img src="images/medium.png" alt="medium"/></a></li>
+          </ul>
+          <div class="ending-line">
+          </div>
+      </footer>
```

Comment:
- [ ] You did well 😃 at adding the styles, however, I see that you didn't stick to the Figma Design at the socials as stated by the previous reviewer, so please try to use the same color for the icons of the social media to be like the Figma Design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/222162923-e6408e4b-15b3-4936-b8b4-5ef23bdc4a3d.png)   |    ![image](https://user-images.githubusercontent.com/83360636/222162817-34d79994-3edd-4da9-82b1-da0a648cd0e2.png)  |